sites/wordpress/search_redirector_override.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * NOTE: this is a bit of a quick-fix solution to resolve a production issue WEB-194.
+ * 
+ * - the following hijacks the search form submission on the home page, preventing 
+ *   a call from being made to https://github.com/bu-ist/bu-library-theme/blob/master/search-redirector.php,
+ *   which, if allowed, results in 403 errors due to 'Web Application Firewall' configurations by IS&T
+ * - it
```

Comment:
stopped in the middle of writing this, apparently 😞 😅. get rid of this bullet or finish the thought.